composer.json

```diff
@@ -33,5 +34,11 @@
   },
   "require-dev": {
     "spryker/code-sniffer": "dev-master"
+  },
+  "scripts": {
```

Comment:
All files should now have 4 space indentation. Please check master, you might have to merge it into here or rebase.